CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## v4.0.2
```

Comment:
@Ikana please do a *rebase* on the latest master rather than clicking "update branch", which creates a merge commit.